erp-server/src/main/resources/application.properties

```diff
@@ -1,22 +1,30 @@
+# DB
 spring.jpa.hibernate.ddl-auto=${SPRING_JPA_HIBERNATE_DDL_AUTO}
-
 spring.datasource.platform=${SPRING_DATASOURCE_PLATFORM}
 spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
 spring.datasource.url=${SPRING_DATASOURCE_URL}?&rewriteBatchedStatements=true
 spring.datasource.username=${SPRING_DATASOURCE_USERNAME}
 spring.datasource.password=${SPRING_DATASOURCE_PASSWORD}
 
 # Batching - https://vladmihalcea.com/how-to-batch-insert-and-update-statements-with-hibernate/
-hibernate.jdbc.batch_size=30
-hibernate.order_inserts=true
-hibernate.order_updates=true
-hibernate.jdbc.batch_versioned_data=true
+spring.jpa.properties.hibernate.jdbc.batch_size=30
+spring.jpa.properties.hibernate.order_inserts=true
+spring.jpa.properties.hibernate.order_updates=true
+spring.jpa.properties.hibernate.jdbc.batch_versioned_data=true
```

Comment:
Good catch. I forgot we decided not to merge the branch where these were fixed previously.